cookiecutter/vcs.py

```diff
@@ -103,6 +103,8 @@ def clone(repo_url, checkout=None, clone_to_dir=".", no_input=False):
         msg = "'{0}' is not installed.".format(repo_type)
         raise VCSNotInstalled(msg)
 
+    if repo_url.endswith('/'):
+        repo_url = repo_url[:-1]
```

Comment:
There are any repo_url starting with a slash? Like: repo_url = '/home/eliasdorneles/cookiecutter-pypackage/'

If so, it would be better to use rstrip(). I will try it and run the tests.